tests/suites/test_suite_debug.function

```diff
@@ -195,3 +195,17 @@ exit:
     mbedtls_ssl_config_free(&conf);
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void check_mbedtls_calloc_overallocation(int num, int size)
```

Comment:
Non-blocker: it would be simpler overall to create `test_suite_platform` in 2.28.